vendor/teleopti/calendar-data-updater.directive.js

```diff
@@ -0,0 +1,159 @@
+(function() {
```

Comment:
I added a js folder where you can put it. 
But how do you include those directives in our project ? Right now, the styleguide exports only css. Maybe it's time to work on how to bundle styleguide js (grunt task...etc), what do you think ?